app/_src/kubernetes-ingress-controller/get-started/key-authentication.md

```diff
@@ -6,3 +6,117 @@ type: tutorial
 purpose: |
   Show how to enable Key Authentication with KIC, including creating a consumer and a credential
 ---
+
+
+Authentication is the process of verifying that a requester has permissions to access a resource. 
+As its name implies, API gateway authentication authenticates the flow of data to and from your upstream services. 
+
+{{site.base_gateway}} has a library of plugins that support 
+the most widely used [methods of API gateway authentication](/hub/#authentication). 
+
+Common authentication methods include:
+* Key Authentication
+* Basic Authentication
+* OAuth 2.0 Authentication
+* LDAP Authentication Advanced
+* OpenID Connect
+
+## Authentication benefits
+
+With {{site.base_gateway}} controlling authentication, requests won't reach upstream services unless the client has successfully
+authenticated. This means upstream services process pre-authorized requests, freeing them from the 
+cost of authentication, which is a savings in compute time *and* development effort.
+
+{{site.base_gateway}} has visibility into all authentication attempts, which provides the ability to build 
+monitoring and alerting capabilities supporting service availability and compliance. 
+
+For more information, see [What is API Gateway Authentication?](https://konghq.com/learning-center/api-gateway/api-gateway-authentication).
+
+### Add authentication to the echo service
+
+1. Create a new `key-auth` plugin:
+
+    ```bash
+    echo "
+    apiVersion: configuration.konghq.com/v1
+    kind: KongPlugin
+    metadata:
+      name: key-auth
+    plugin: key-auth
+    config:
+      key_names:
+      - apikey
+    " | kubectl apply -f -
+    ```
+
+1. Apply the `key-auth` plugin to the `echo` service:
+
+    ```bash
+    kubectl annotate service echo konghq.com/plugins=key-auth
```

Comment:
I get this error when I try this after rate limiting: ```error: --overwrite is false but found the following declared annotation(s): 'konghq.com/plugins' already has a value (rate-limit-5-min)
rajakavitha@Rajakavithas-MacBook-Air ~ % kubectl annotate service echo konghq.com/plugins=key-auth --overwrite

service/echo annotated
rajakavitha@Rajakavithas-MacBook-Air ~ % HTTP/1.1 401 Unauthorized
Date: Wed, 11 Jan 2044 18:33:46 GMT
Content-Type: application/json; charset=utf-8
WWW-Authenticate: Key realm="kong"
Content-Length: 45
X-Kong-Response-Latency: 1
Server: kong/3.4.2

{
  "message":"No API key found in request"
}

zsh: no such file or directory: HTTP/1.1
zsh: command not found: Date:
zsh: command not found: Content-Type:
zsh: command not found: WWW-Authenticate:
zsh: command not found: Content-Length:
zsh: command not found: X-Kong-Response-Latency:
zsh: command not found: Server:
zsh: command not found: message:No API key found in request
rajakavitha@Rajakavithas-MacBook-Air ~ % kubectl create secret generic alex-key-auth \
  --from-literal=kongCredType=key-auth \
  --from-literal=key=hello_world

secret/alex-key-auth created
rajakavitha@Rajakavithas-MacBook-Air ~ % echo "apiVersion: configuration.konghq.com/v1
kind: KongConsumer
metadata:
  name: alex
  annotations:
    kubernetes.io/ingress.class: kong
username: alex
credentials:
- alex-key-auth
" | kubectl apply -f -

kongconsumer.configuration.konghq.com/alex created
rajakavitha@Rajakavithas-MacBook-Air ~ % curl -H 'apikey: hello_world' $PROXY_IP/echo

Welcome, you are connected to node orbstack.
Running on Pod echo-965f7cf84-wqhp9.
In namespace default.
With IP address 192.168.194.10.
``` with the --overwrite option the instructions work flawlessly